src/components/App.tsx

```diff
@@ -2261,6 +2266,21 @@ class App extends React.Component<AppProps, AppState> {
         });
       } else if (data.text) {
         const maybeUrl = extractSrc(data.text);
+        if (
+          !isPlainPaste &&
+          embeddableURLValidator(maybeUrl, new RegExp(RE_GDOCS))
+        ) {
+          const embeddable = this.insertEmbeddableElement({
+            sceneX,
+            sceneY,
+            link: normalizeLink(maybeUrl),
+          });
+          if (embeddable) {
+            this.setState({ selectedElementIds: { [embeddable.id]: true } });
+          }
+          return;
+        }
```

Comment:
what is this for?

Reply:
Sorry, forgot to remove these changes. I was trying to add docs image as an embeddable element. 
Removed Now :+1: